crates/frontend/src/pages/default_config.rs

```diff
@@ -228,8 +278,15 @@ pub fn default_config() -> impl IntoView {
                     }
                 }}
                 {move || {
-                    let default_config = default_config_resource.get().unwrap_or(vec![]);
+                    let default_config = default_config_resource
+                        .get()
+                        .unwrap_or(PaginatedResponse {
+                            total_items: 1,
```

Comment:
here default items and page is 1, in other places it is 0

Reply:
I think it should be 1 as default as 0 makes no sense here